src/paper/ingestion/clients/enrichment/bluesky.py

```diff
@@ -0,0 +1,235 @@
+import logging
+from typing import Dict, List, Optional
+
+from atproto import Client
+from django.conf import settings
+
+from ..base import RateLimiter
+
+logger = logging.getLogger(__name__)
+
+
+class BlueskyClient:
+    """
+    Client for interacting with the Bluesky API.
+    Uses the AT Protocol to search for and retrieve posts mentioning papers.
+    Handles authentication, rate limiting, and error handling.
+    """
+
+    DEFAULT_RATE_LIMIT = 10.0
+    MAX_SEARCH_RESULTS = 100
+
+    def __init__(
+        self,
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+        client: Optional[Client] = None,
+        rate_limit: float = DEFAULT_RATE_LIMIT,
+    ):
+        """
+        Initialize the Bluesky client.
+
+        Args:
+            username: Bluesky username (handle). Defaults to settings.BLUESKY_USERNAME
+            password: Bluesky app password. Defaults to settings.BLUESKY_PASSWORD
+            client: AT Protocol client instance. If None, creates a new Client()
+            rate_limit: Maximum requests per second (default: 10.0)
+        """
+        self.username = username or getattr(settings, "BLUESKY_USERNAME", "")
+        self.password = password or getattr(settings, "BLUESKY_PASSWORD", "")
+        self.rate_limiter = RateLimiter(rate_limit)
+        self.client = client or Client()
+        self.authenticated = False
+        self._authenticate()
+
+    def _authenticate(self) -> bool:
```

Comment:
The return value is misleading, since it will only return `True` or raise an exception.
```suggestion
    def _authenticate(self):
```